docs/_conversion/maintaining.md

```diff
@@ -0,0 +1,140 @@
+
+![](https://cdn.zappy.app/27b0a08799b047fc9db1ed204d030377.gif)
+
+>Pro Tip: A handy way to understand this is to have a look at the [schema](https://zapier.github.io/zapier-platform-schema/build/schema.html) of the integration defintion you're building.  [Here](https://zapier.github.io/zapier-platform-schema/build/schema.html#basicpollingoperationschema) you'll see the structure of the definion of a polling trigger.  Notice that the `perform` field takes a request configuration object, or it simply takes a Javascript function.  The new builder UI reflects this.  When you configure an API interaction, you use the form UI to configure a request, or you use "Code Mode" to write a function to be called instead.
+
+**No pre or post functions**
+
+In Web Builder, you could write custom code that ran before the request was made, or after the request returned, or you could take over complete control of makeing the request and returning data to Zapier.  In the new platform there is only the 'complete control' option.  Your request can implement a `perform` function that takes a bundle as input and is expected to return data when the request handling is completed.  
+
+**Promise based API**
+
+A big difference in the API of the new environment is that the request API is ansyncronous, using promises rather than syncronous requests or callbacks.  For developers new to Javascript promises this can be a small stumbling block.  We recommend spending a few minutes getting familiar with how promises work in general.  The `z.request` library works very similarly to the promise-based Fetch API, so most articles and tutorials about that apply to working with custom code in Zapier's environment.  
+
+- [Using promises (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)
+- [JavaScript Promises in Depth (Egghead.io)](https://egghead.io/courses/javascript-promises-in-depth)
+- [Using Fetch (MDN)](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
+- [A re-introduction to JavaScript (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript)
+
+**No libraries included**
+
+The legacy Web Builder tool included several Javascript libraries, such as Lodash, in its context that you could use in your script code.  Those are no longer available when writing scripting code in the new platform's UI tool.  You'll be able to use any Javacript feature of Node.js (version 12 at the time of this writing, possibly v14 by the time you're reading this), including its standard library with `z.require`.
```

Comment:
Strictly speaking it was `underscore`, not lodash. Also, I'd mention `jquery` - that was a popular one.